### DISCUSSION TITLE
Option for Windows 11 Command Bar (classic Address Bar) doesn't work as intended. Is there a way to fix this and how?

### DISCUSSION THREAD

**Discussion by Haxogone** [valinet/ExplorerPatcher#3317] [Q&A]
It just behaves like the default Windows 11 address bar.
<img width="824" alt="command bar image" src="https://github.com/valinet/ExplorerPatcher/assets/110359136/3bfa8227-39cf-4b0c-a6e5-fa51c01f51da">

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2701

please close this.
